Source/Core/Core/PowerPC/MMU.cpp

```diff
@@ -938,52 +938,42 @@ bool MMU::IsOptimizableRAMAddress(const u32 address, const u32 access_size) cons
 }
 
 template <XCheckTLBFlag flag>
-bool MMU::IsRAMAddress(u32 address, bool translate)
+bool MMU::IsEffectiveRAMAddress(u32 address)
 {
-  if (translate)
+  if (auto translate_address = TranslateAddress<flag>(address); translate_address.Success())
   {
-    auto translate_address = TranslateAddress<flag>(address);
-    if (!translate_address.Success())
-      return false;
     address = translate_address.address;
```

Comment:
Can't this bit be removed as well as `translate_address.address` is passed directly to `IsPhysicalRAMAddress`?